client/test/redux/__tests__/mealAction/meal-test.js

```diff
@@ -0,0 +1,149 @@
+import configureMockStore from 'redux-mock-store';
```

Comment:
Kevin. There are many issues with this PR. You can't push your code when your CIs are not passing and I have told you this before. Learn to listen and pay attention to details. Your mistakes are becoming unbearable because you are slowing your team down.